File: 167-two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int[] result = {0, 0};

        int r = numbers.length -1;
        int l = 0;
        while (l < r)
        {
            int sum = numbers[l] + numbers[r];

            if(sum == target)
            {
                result[0] = l+1;
                result[1] = r+1;
                return result;
            }
            if(sum > target)
                r--;
            else
                l++;

        }
        return result;
    }
}

File: 167-two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        r = len(numbers)-1
        l = 0
        while(l < r):
            sum = numbers[r] + numbers[l]
            if(target == sum):
                return[l+1,r+1]
            elif (sum < target):
                l+=1
            else:
                r-=1

        return[0,0]